### DISCUSSION TITLE
cva() responsive variants?

### DISCUSSION THREAD

**Discussion by traviswimer** [chakra-ui/panda#1413] [Q&A] [answered]
### Subject

cva() recipes

### Description

On the "Recipes" page under [Limitations](https://panda-css.com/docs/concepts/recipes#limitations), the docs say this:

> Recipes created from `cva` cannot have responsive or conditional values. Only layer recipes can have responsive or conditional values.

But if you try to use breakpoints in your `cva` variants, it does seem to output CSS with the correct media queries. For example:

```typescript
const myRecipe = cva({
  base: {
    width: "100%",
  },
  variants: {
    type: {
      something: {
        md: {
          width: "50%",
        },
      },
      somethingElse: {
        md: {
          width: "75%",
        },
      },
    },
  },
});
```

Am I misunderstanding what "responsive" means here or is the documentation incorrect?

**Comment by segunadebayo** [comment, marked as answer]
If you write the responsive styles within the variant, it should work as expected.

What we mean in the docs is if you do:

```js
myRecipe({ type: { md: "something", lg: "somethingElse" } })
```

This will not work.

**Reply by traviswimer** [reply]
Thanks! That makes sense.
